server/src/main/resources/transport/upper_bounds/8.18.csv

```diff
@@ -1 +1 @@
-initial_elasticsearch_8_18_8,8840010
+transform_check_for_dangling_tasks,8840011
```

Comment:
weird that CI pushed this changes - probably merging main here will make these disappear